src/Symfony/Component/AssetMapper/Tests/Compiler/JavaScriptImportPathCompilerTest.php

```diff
@@ -93,6 +93,48 @@ public function testCompileFindsCorrectImports(string $input, array $expectedJav
 
     public static function provideCompileTests(): iterable
     {
+        yield 'import_in_multiline_comment' => [
+            'input' => <<<EOF
+// Asynchronous loading
+/**
+ * Used in conjunction with the `m` component to reduce bundle size.
+ *
+ * `m` is a version of the `motion` component that only loads functionality
+ * critical for the initial render.
+ *
+ * `LazyMotion` can then be used to either synchronously or asynchronously
+ * load animation and gesture support.
+ *
+ * ```jsx
+ * // Synchronous loading
+ * import { LazyMotion, m, domAnimations } from "framer-motion"
+ *
+ * function App() {
+ *   return (
+ *     <LazyMotion features={domAnimations}>
+ *       <m.div animate={{ scale: 2 }} />
+ *     </LazyMotion>
+ *   )
+ * }
+ *
+ * // Asynchronous loading
+ * import { LazyMotion, m } from "framer-motion"
+ *
+ * function App() {
+ *   return (
+ *     <LazyMotion features={() => import('./path/to/domAnimations')}>
+ *       <m.div animate={{ scale: 2 }} />
+ *     </LazyMotion>
+ *   )
+ * }
+ * ```
+ *
+ * @public
+ */
+EOF,
+            'expectedJavaScriptImports' => [],
+        ];
+
```

Comment:
Nice catch, @tacman! 
Could you reduce this test to the minimal amount of code? It will be easier to later maintain and also will ensure we are testing this case and not any other conditions. :)